src/youtube/js/youtube.js

```diff
@@ -86,6 +86,7 @@ require("../../../html5-common/js/utils/constants.js");
    * @property {object} youtubePlayer The youtube player is creted.
    */
   function onYouTubeIframeAPIReady() {
+    if ( youtubeID == null || youtubePlayer ) { return};
```

Comment:
@aeng7 In this.destroy(), setVideoUrl() is called with empty string. Hence added this check here.